File: server/iih_pub/iih.ci_pub/src/main/java/iih/ci/ord/i/ems/ICiEmsMainService.java
package iih.ci.ord.i.ems;

import iih.ci.ord.d.ems.ems.EmsCrtDTO;
import iih.ci.ord.d.ems.ems.EmsLoadDTO;
import iih.ci.ord.d.ems.ems.EmsRstDTO;
import iih.ci.ord.d.ems.ems.EmsSaveDTO;
import xap.mw.core.data.BizException;

/**
 * 医疗单主服务接口
 * @author wangqingzhu
 *
 */
public interface ICiEmsMainService {

	/**
	 * 创建医疗单UI对象
	 * @param ems
	 * @return
	 */
	public abstract EmsRstDTO[] create(EmsCrtDTO[] emsarray) throws BizException ;
	
	/**
	 * 加载医疗单
	 * @param ems
	 * @return
	 */
	public abstract EmsRstDTO[] load(EmsLoadDTO[] emsarray) throws BizException ;
	
	/**
	 * 保存医疗单
	 * @param ems
	 * @return
	 */
	public abstract EmsRstDTO save(EmsSaveDTO ems) throws BizException ;
}

File: client/iih_stub/iih.ci_stub/iih.ci.ord/i/ICiEmsMainService.cs
using iih.ci.ord.dto.emsmain;
using iih.ci.ord.ciorder.d;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace iih.ci.ord.i
{
    public interface ICiEmsMainService
    {
        /**
	 * 创建医疗单UI对象
	 * @param ems
	 * @return
	 */
        EmsRstDTO[] create(EmsCrtDTO[] ems)  ;

        /**
         * 加载医疗单
         * @param ems
         * @return
         */
        EmsRstDTO[] load(EmsLoadDTO[] ems);



        /**
         * 保存医疗单
         * @param ems
         * @return
         */
        EmsRstDTO save(EmsSaveDTO ems);
    }
}
